templates/docker-entrypoint.sh

```diff
@@ -74,24 +81,25 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
   : "${ROUNDCUBEMAIL_SKIN:=elastic}"
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
   : "${ROUNDCUBEMAIL_REQUEST_PATH:=/}"
+  : "${ROUNDCUBEMAIL_COMPOSER_PLUGINS_FOLDER:=$INSTALLDIR}"
 
   if [ ! -z "${ROUNDCUBEMAIL_INSTALL_PLUGINS}" ]; then
```

Comment:
Can't we just check if `ROUNDCUBEMAIL_COMPOSER_PLUGINS` is non-empty?
In what case would somebody set `ROUNDCUBEMAIL_COMPOSER_PLUGINS` but `ROUNDCUBEMAIL_INSTALL_PLUGINS=0`

Reply:
Good idea, I changed this and added an example